src/components/autocomplete/autocomplete.tsx

```diff
@@ -152,6 +152,15 @@ export class Autocomplete<Value, Suggestion> extends React.Component<Autocomplet
 
     renderChips() {
         const { items, onDelete } = this.props;
+
+        /**
+         * If input startAdornment prop is not undefined, input's label will stay above the input.
+         * If there is not items, we want the label to go back to placeholder position.
+         */
+        if (items.length === 0) {
+            return;
+        }
+
```

Comment:
not necessary condition :) if items is empty nothing will render anyway

Reply:
See a comment above the condition.